### PR TITLE
PCHR-1862: Fix date picker text getting clipped and toil expiry not aligned

### PR DESCRIPTION
**Pull request by dhavalsharma**
Reduce font size of date text in date picker to prevent its clipping. This is a temporary solution until theme is normalized to adjust date picker text itself.

image

## expiry date for span is aligned correctly
image

### CODE REVIEW

**Review by AkA84 — CHANGES_REQUESTED** [review]
can we fix this by using bootstrap grid (`col-sm-6`, etc), instead of ad-hoc magic numbers?

**Comment by dhavalsharma** [comment]
grid ones are not working , either its becoming too long and displacing the other dropdown or too small to clip date

**Comment by AkA84** [comment]
@dhavalsharma can you show a screenshot of that? not sure how it can displaced the other dropdowns if you adjust all the grid cells accordingly

**Comment by dhavalsharma** [comment]
with xs-5 it leaves lots of white space
image

File: civihr_default_theme/scss/components/_leave-request-modal.scss [hunk 1]
```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
     }
   }
 }
+
+/*
+* This is a TEMP fix to fit the text of input controls like date picker within input box
+* It is hardcoded to 13px for now.
+*/
+.chr_leave-request-modal__input-font-size {
```
Comment (AkA84) [inline civihr_default_theme/scss/components/_leave-request-modal.scss]: this is just a hack, we don't need an entire new selector, can't you just reference classes/tags that are already there? this is a bit overkill

File: civihr_default_theme/scss/components/_leave-request-modal.scss [hunk 2]
```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
...
+  font-size: 13px;
+}
+
+.chr_leave-request-modal__span-expiry {
```
Comment (AkA84) [inline civihr_default_theme/scss/components/_leave-request-modal.scss]: This looks an ad-hoc fix, did you see what the reason is for the misalingment in the first place

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/components/_leave-request-modal.scss (hunk 1 above)
Comment (dhavalsharma) [reply civihr_default_theme/scss/components/_leave-request-modal.scss]: do you mean something like `chr_leave-calendar__legend__label` but its name suggests its for calendar?

File: civihr_default_theme/scss/components/_leave-request-modal.scss (hunk 2 above)
Comment (dhavalsharma) [reply civihr_default_theme/scss/components/_leave-request-modal.scss]: It is combination of `control-label` and `label` which caused the text to be mis aligned after addition of `uib-datepicker-popup` for expiry date. 
If i use `form-control` in span it gives a border rectangle but aligns it well. is that ok?

**Review by AkA84 — APPROVED** [review]
@dhavalsharma approved, but squash the PR review commits

**Pull request merged**